k8s/spinnaker/README.md

```diff
@@ -10,97 +10,112 @@ If you would like to learn more about Spinnaker, please visit the
 
 Popular open stacks on Kubernetes packaged by Google.
 
+## About Spinnaker for Google Cloud Platform
+
+This solution installs a single instance of Spinnaker into a GKE cluster in a
+production-ready configuration. The installation follows recommended practices
+for running Spinnaker on Google Cloud Platform, and is integrated with related
+Google Cloud services, such as [Cloud Build](https://cloud.google.com/cloud-build/).
+
+This solution also provides a simplified configuration experience, as well as a
+management environment and workflow for ongoing administration of Spinnaker.
+
 ## Architecture
 
 ### TODO: Create architecture diagram
 
 ![Architecture diagram](resources/spinnaker-k8s-app-architecture.png)
 
-This solution install a single instance of Spinnaker into a Kubernetes cluster.
-Our goal is to provide an easy initial installation experience, as well as an
-environment and workflow for ongoing management of Spinnaker by an individual
-or a team.
-
 Spinnaker is composed of a number of individual
-[micro services](https://www.spinnaker.io/reference/architecture/), and each
-will be deployed in their own Pods, managed by Deployment objects, behind
-Service objects.
+[microservices](https://www.spinnaker.io/reference/architecture/). These
+will be deployed in their own Kuberetes Pods, managed by Deployment objects,
+behind Service objects.
 
 [Halyard](https://www.spinnaker.io/reference/halyard/) is Spinnaker's
 configuration service and consists of a CLI and a daemon. The CLI will be
-installed in your Cloud Shell and the daemon will be installed in a Pod managed
-by a StatefulSet object.
+installed in the management environment (based on Cloud Shell) included in
+this solution. The daemon will be installed in a Pod managed by a StatefulSet
+object.
 
-[Spin](https://www.spinnaker.io/guides/spin/app/) is Spinnaker's CLI. It will
-be installed in your Cloud Shell.
+[Spin](https://www.spinnaker.io/guides/spin/app/) is Spinnaker's CLI. It is also
```

Comment:
We link above to the page on spinnaker.io that describes all the microservices, and here we call out two of them specifically. I suggest adding a sentence above this line that says something like "Two of these microservices are important for our puposes here." And then set off Halyard and Spin with bullets.

Reply:
Instead, add, below that first para, "We provide the following two components to help you manage your Spinnaker instance.